docs/getting-started/setup/editor-types.md

```diff
@@ -0,0 +1,83 @@
+---
+category: setup
+menu-title: Editor types
+meta-title: Editor types | CKEditor 5 documentation
+meta-description: Learn about available editor types.
+order: 25
+modified_at: 2024-06-06
+---
+# Editor types
+
+The editor's user interface is mostly dependent on the editor types. Functionality of the editor is provided by specialized features, accessible via a configurable toolbar or keyboard shortcuts. Some of these features are only available with certain editor types.
+
+<info-box>
+	All of these elements can most easily be configured with [CKEditor&nbsp;5 Builder](https://ckeditor.com/ckeditor-5/builder?redirect=docs).
+</info-box>
+
+There are six ready-made editor types (see below) available for CKEditor&nbsp;5. They offer different functional approach to editing as well as different UI solutions. Editor types are imported from the main `ckeditor5` package, the same way features are imported, as shown in the {@link getting-started/quick-start Quick start} guide.
+
+Other, custom-taylored editor types can be made using the {@link framework/index CKEditor&nbsp;5 Framework}.
```

Comment:
Could link to more specific https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/framework/deep-dive/ui/external-ui.html

Reply:
Sure. It will also be linked back here, by the way ;)